src/constants.js

```diff
@@ -0,0 +1,60 @@
+export const TASKS_MANAGEMENT_MAIN_MENU_CONTRIBUTION_KEY = 'tasksManagement.MainMenu';
+
+export const DEFAULT_DEBOUNCE_TIME = 500;
+export const DEFAULT_PAGE_SIZE = 10;
+export const EMPTY_STRING = '';
+export const HYPHEN = '-';
+export const ROWS_PER_PAGE_OPTIONS = [10, 20, 50, 100];
+export const CONTAINS_LOOKUP = 'Icontains';
+
+export const TASK_STATUS = {
+  RECEIVED: 'RECEIVED',
+  ACCEPTED: 'ACCEPTED',
+  COMPLETED: 'COMPLETED',
+  FAILED: 'FAILED',
+};
+
+export const POLICY_STATUS = {
+  ALL: 'ALL',
+  ANY: 'ANY',
+  N: 'N',
+};
+
+export const TASK_COLUMNS = {
+  SOURCE: 'source',
+  TYPE: 'type',
+  ENTITY: 'entity',
+  ASSIGNEE: 'assignee',
+  BUSINESS_STATUS: 'businessStatus',
+  STATUS: 'status',
+};
+
+export const TASK_PREVIEW_COLUMNS = [
+  TASK_COLUMNS.SOURCE,
+  TASK_COLUMNS.TYPE,
+  TASK_COLUMNS.ENTITY,
+  TASK_COLUMNS.ASSIGNEE,
+  TASK_COLUMNS.BUSINESS_STATUS,
+  TASK_COLUMNS.STATUS,
+];
+
+export const TASK_STATUS_LIST = [
+  TASK_STATUS.RECEIVED, TASK_STATUS.ACCEPTED, TASK_STATUS.COMPLETED, TASK_STATUS.FAILED];
+
+export const POLICY_STATUS_LIST = [POLICY_STATUS.ALL, POLICY_STATUS.ANY, POLICY_STATUS.N];
+
+// RIGHTS HAVE TO BE CHANGED AFTER BE IMPLEMENTATION
+export const RIGHT_TASKS_MANAGEMENT_SEARCH = 159001;
+
+export const BENEFIT_PLAN_TASKS_SEARCH = 180001;
+export const BENEFIT_PLAN_TASKS_CREATE = 180001;
+export const BENEFIT_PLAN_TASKS_UPDATE = 180001;
+export const BENEFIT_PLAN_TASKS_DELETE = 180001;
```

Comment:
Benefit Plan Tasks Rights are the same as ones for the TASK_GROUP_SEARCH. Please change it, also those should be moved to the Beneiftplan frontend module along with BenefitPlanTask components.

Reply:
Moving all benefit plan logic to the other modules will be handled in scope of the [CM-139](https://openimis.atlassian.net/browse/CM-139).